sympy/integrals/rubi/parsetools/parse.py

```diff
@@ -327,10 +327,12 @@ def generate_sympy_from_parsed(parsed, wild=False, symbols=[], replace_Int=False
 
     return out
 
-def get_free_symbols(s, symbols, free_symbols=[]):
+def get_free_symbols(s, symbols, free_symbols=None):
     '''
     Returns free_symbols present in `s`.
     '''
+    if not free_symbols:
```

Comment:
FWIW, I like to write this `free_symbols = free_symbols or []`.

Reply:
I never really understood that construction fully. Or rather, I think I get it when you write it, but still a bit away from my everyday toolbox. Will change it though (so I may start to use it).

Reply:
if the first `or` argument is false you get the second: `0 or 2 -> 2`. The result is not a boolean, the result is one of the arguments.